spec/integration/api/post_interactions_controller_spec.rb

```diff
@@ -396,7 +396,7 @@
       post(
         api_v1_post_vote_path(@poll_post.guid),
         params: {
-          poll_answer_id: -1,
+          poll_answer: -1,
```

Comment:
Layout/AlignHash: Align the elements of a hash literal if they span more than one line.